recipes/taxor/meta.yaml

```
@@ -0,0 +1,28 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: taxor
+  version: {{ version }}
+
+source:
+  url: https://github.com/JensUweUlrich/Taxor/releases/download/{{ version }}/taxor.tar.gz # [linux]
+  md5: 8e5476aadc3fecab580e8b1b73ba3dff # [linux]
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('taxor', max_pin='x') }}
```

Since the version is `0.x`:
```suggestion
    - {{ pin_subpackage('taxor', max_pin='x.x') }}
```